haskell-language-server.cabal

```diff
@@ -454,10 +463,7 @@ test-suite func-test
 
   default-extensions: OverloadedStrings
   ghc-options:
-    -Wall -Wno-name-shadowing -threaded -rtsopts -with-rtsopts=-N -Wno-unticked-promoted-constructors
-
-  if flag(pedantic)
-    ghc-options: -Werror -Wredundant-constraints
```

Comment:
This configuration changes compared to before. E.g. `-Wredundant-constraints` is always added to the list of warnings.